docs/pages/typography-helpers.md

```diff
@@ -49,6 +51,22 @@ Lighten up your headers by adding a class of `.subheader` to any header element.
 
 ---
 
+## Drop Caps
+
+Add a drop cap of 3 lines to the begining of a paragraph.
```

Comment:
I do not think this is necessary to precise _how a dropcap is generally used_. But maybe _what is it_ would be useful:

> Add a drop cap of 3 lines to the begining of a paragraph. A dropcap (or _Initial_) is a letter at the beginning of a word, a chapter, or a paragraph that is larger than the rest of the text. 

Thanks Wikipédia